[2024-06-24 15:06:14,272] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2024-05-20T00:00:00+00:00 [queued]>
[2024-06-24 15:06:14,348] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task 2024-05-20T00:00:00+00:00 [queued]>
[2024-06-24 15:06:14,353] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 15:06:14,355] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 15:06:14,356] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 15:06:14,412] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2024-05-20T00:00:00+00:00
[2024-06-24 15:06:14,424] {standard_task_runner.py:52} INFO - Started process 34862 to run task
[2024-06-24 15:06:14,436] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'second_task', '2024-05-20T00:00:00+00:00', '--job-id', '11805', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpqi_axfjg', '--error-file', '/tmp/tmpz00vmieq']
[2024-06-24 15:06:14,444] {standard_task_runner.py:77} INFO - Job 11805: Subtask second_task
[2024-06-24 15:06:14,548] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.second_task 2024-05-20T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 15:06:14,718] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2024-05-20T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-20T00:00:00+00:00
[2024-06-24 15:06:14,724] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-06-24 15:06:14,726] {bash.py:158} INFO - Running command: echo hey, I am task2 and will be running after task1!
[2024-06-24 15:06:14,743] {bash.py:169} INFO - Output:
[2024-06-24 15:06:14,745] {bash.py:173} INFO - hey, I am task2 and will be running after task1!
[2024-06-24 15:06:14,747] {bash.py:177} INFO - Command exited with return code 0
[2024-06-24 15:06:14,820] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20240520T000000, start_date=20240624T150614, end_date=20240624T150614
[2024-06-24 15:06:14,907] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24 15:06:14,930] {local_task_job.py:146} INFO - Task exited with return code 0
